app/views/orders/_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <br>
   <div class="form-group">
       <%= form.label :travelers, class:'form-label mb-2 mt-3' %><br />
-      <%= form.select :travelers,User.all.traveler.pluck(:first_name),{prompt:'Select'},{class: "form-control"}%>
+      <%= form.select :travelers,User.traveler.pluck(:first_name),{prompt:'Select'},{class: "form-control"}%>
```

Comment:
use instance variable here, query in view is not good approach.
add this line in controller and use @users variable in view
`@users  = User.all.traveler.pluck(:first_name)`

Reply:
i have modified this file already. Will push the code after changes completed